systemd/README.md

```diff
@@ -0,0 +1,207 @@
+* Paste the following:
+
+```
+[Unit]
+Description=go-livepeer transcoder daemon service
+After=network.target
+
+[Service]
+User=livepeer
+Group=livepeer
+Type=simple
+Restart=always
+RestartSec=90s
+WorkingDirectory=/home/livepeer/go-livepeer/
+ExecStart=/home/livepeer/go-livepeer/livepeer -network rinkeby -transcoder -orchAddr 127.0.0.1:8935 -orchSecret osecret.txt -nvidia 0
+
+[Install]
+WantedBy=default.target
+```
+
+Update the `ExecStart` line to match your configuration. To use offline mode rather than the rinkeby testnet, replace the `ExecStart` line with the following:
+
+```
+ExecStart=/home/livepeer/go-livepeer/livepeer -network offchain -transcoder -orchAddr 127.0.0.1:8935 -orchSecret osecret.txt -nvidia 0
+```
+
+* Save the document and exit `vi` by first hitting the `ESC` key, then `:` followed by `wq` and `<ENTER>`
+
+* Enable and start the services:
+
+```bash
+systemctl enable transcoder.service
+systemctl start transcoder.service
+```
+
+* To monitor the service:
+
+```bash
+journalctl -f --unit=transcoder.service
+```
+
+To run the orchestrator and transcoder as a single combined service:
+
+* Create the `orchtran.service` file:
+
+```bash
+vi /etc/systemd/system/orchtran.service
+```
+
+* Push the `i` key to get into insertion mode.
+
+* Paste the following:
+
+```
+[Unit]
+Description=go-livepeer orchestrator+transcoder daemon service
+After=network.target
+
+[Service]
+User=livepeer
+Group=livepeer
+Type=simple
+Restart=always
+RestartSec=90s
+WorkingDirectory=/home/livepeer/go-livepeer/
+ExecStart=/home/livepeer/go-livepeer/livepeer -network rinkeby -orchestrator -transcoder -pricePerUnit 1 -nvidia 0 -initializeRound=true -serviceAddr=127.0.0.1:8935
+
+[Install]
+WantedBy=default.target
+```
+
+Update the `ExecStart` line to match your configuration. To use offline mode rather than the rinkeby testnet, replace the `ExecStart` line with the following:
+
+```
+ExecStart=/home/livepeer/go-livepeer/livepeer -network offchain -orchestrator -transcoder -serviceAddr 127.0.0.1:8935 -orchAddr 0.0.0.0:8935 -nvidia 0
+```
+
+* Save the document and exit `vi` by first hitting the `ESC` key, then `:` followed by `wq` and `<ENTER>`
+
+* Enable and start the services:
+
+```bash
+systemctl enable orchtran.service
+systemctl start orchtran.service
+```
+
+* To monitor the service:
+
+```bash
+journalctl -f --unit=orchtran.service
+```
+
+To run a broadcaster node service:
+
+* Create the `broadcaster.service` file:
+
+```bash
+vi /etc/systemd/system/broadcaster.service
+```
+
+* Push the `i` key to get into insertion mode.
+
+* Paste the following:
+
+```
+[Unit]
+Description=go-livepeer broadcaster daemon service
+After=network.target
+
+[Service]
+User=livepeer
+Group=livepeer
+Type=simple
+Restart=always
+RestartSec=90s
+WorkingDirectory=/home/livepeer/go-livepeer/
+ExecStart=/home/livepeer/go-livepeer/livepeer -network rinkeby -broadcaster -orchAddr 127.0.0.1:8935 -cliAddr 127.0.0.1:7936 -httpAddr 127.0.0.1:8936 -depositMultiplier 1
```

Comment:
This is kinda nitpicking here but `-cliAddr` defaults to `127.0.0.1` by default, so to rebind the port we can omit the IP and write `-cliAddr :7936` .

Reply:
thanks - updated in 3ebf9056b296713c1f8864640fa186d3ddbcd3ee

Reply:
I made the change, but again, I'm not sure that this is best practice.  The behavior is changed, and it wont work in all network setups.

before:
```
broadcaster_1    | I0107 18:43:27.437937       1 webserver.go:68] CLI server listening on 0.0.0.0:7936
```
after:
```
broadcaster_1    | I0107 18:44:11.013639       1 webserver.go:68] CLI server listening on 127.0.0.1:7936
```

Reply:
That seems strange that when explicitly setting `cliAddr` to 127.0.0.1:7936 that it would rebind to 0.0.0.0 ?

Reply:
right.  very strange.  perhaps I made a mistake - I will retest that today.

Reply:
OK - In my case, I had cliAddr set to 0.0.0.0:793x.  Why is 127.0.0.1 the default?   Shouldn't it be 0.0.0.0 ?
```diff
@@ -2,7 +2,7 @@ version: '3.5'
 services:
   orchestrator:
     image: livepeer/go-livepeer:master
-    command: '-orchestrator --network rinkeby -orchSecret test -serviceAddr orchestrator:8935 -orchAddr 0.0.0.0 -pricePerUnit 1 -initializeRound=true -ethPassword=/root/pw.txt -cliAddr 0.0.0.0:7935 -monitor=true'
+    command: '-orchestrator --network rinkeby -orchSecret test -serviceAddr orchestrator:8935 -pricePerUnit 1 -initializeRound=true -ethPassword=/root/pw.txt -cliAddr :7935 -monitor=true'
     ports:
       - 7935:7935
       - 8935:8935
@@ -21,7 +21,7 @@ services:
       - orchestrator
       - transcoder
     image: livepeer/go-livepeer:master
-    command: '-broadcaster -network rinkeby -rtmpAddr broadcaster -orchAddr orchestrator:8935 -cliAddr broadcaster:7936 -httpAddr broadcaster:8936 -depositMultiplier 1 -ethPassword=/root/pw.txt -cliAddr 0.0.0.0:7936 -monitor=true'
+    command: '-broadcaster -network rinkeby -rtmpAddr broadcaster -orchAddr orchestrator:8935 -cliAddr broadcaster:7936 -httpAddr broadcaster:8936 -ethPassword=/root/pw.txt -cliAddr :7936 -monitor=true'
     ports:
       - 1935:1935
       - 7936:7936
```

Reply:
@mkrufky
> OK - In my case, I had cliAddr set to 0.0.0.0:793x. Why is 127.0.0.1 the default? Shouldn't it be 0.0.0.0 ?

I think it is good security practice - not to expose such sensitive endpoint to outside world by default?

Reply:
That makes sense - agreed.   Thanks.

Reply:
I agree with Ivan here.